File: FacturaElectronicaCR_CS/Clases/ClasesJson.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using Newtonsoft.Json;

namespace FacturaElectronicaCR_CS
{
    class ClasesJson
    {
    }

    class RespuestaHacienda
    {
        [JsonProperty("clave")]
        public string clave { get; set; }

        [JsonProperty("fecha")]
        public string fecha { get; set; }

        [JsonProperty("ind-estado")]
        public string ind_estado { get; set; }

        [JsonProperty("respuesta-xml")]
        public string respuesta_xml { get; set; }
    }

    class Token
    {
        [JsonProperty("access_token")]
        public string access_token { get; set; }

        [JsonProperty("refresh_token")]
        public string refresh_token { get; set; }
    }

    class Recepcion
    {
        [JsonProperty("clave")]
        public string clave { get; set; }

        [JsonProperty("fecha")]
        public string fecha { get; set; }

        [JsonProperty("emisor")]
        public Emisor emisor { get; set; }

        [JsonProperty("receptor")]
        public Receptor receptor { get; set; }

        [JsonProperty("comprobanteXml")]
        public string comprobanteXml { get; set; }
    }

    class Emisor
    {
        [JsonProperty("TipoIdentificacion")]
        public string TipoIdentificacion { get; set; }

        [JsonProperty("numeroIdentificacion")]
        public string numeroIdentificacion { get; set; }
    }

    class Receptor {
        [JsonProperty("TipoIdentificacion")]
        public string TipoIdentificacion { get; set; }

        [JsonProperty("numeroIdentificacion")]
        public string numeroIdentificacion { get; set; }
    }
}

File: FacturaElectronicaCR_VB/Clases/ClasesJson.vb
Imports Newtonsoft.Json
Imports System

Public Class ClasesJson
End Class

Public Class RespuestaHacienda
    <JsonProperty("clave")>
    Public Property clave As String

    <JsonProperty("fecha")>
    Public Property fecha As String

    <JsonProperty("ind-estado")>
    Public Property ind_estado As String

    <JsonProperty("respuesta-xml")>
    Public Property respuesta_xml As String
End Class

Public Class Token
    <JsonProperty("access_token")>
    Public Property access_token As String

    <JsonProperty("refresh_token")>
    Public Property refresh_token As String
End Class

Public Class Recepcion
    <JsonProperty("clave")>
    Public Property clave As String

    <JsonProperty("fecha")>
    Public Property fecha As String

    <JsonProperty("emisor")>
    Public Property emisor As Emisor

    <JsonProperty("receptor")>
    Public Property receptor As Receptor

    <JsonProperty("comprobanteXml")>
    Public Property comprobanteXml As String
End Class

Public Class Emisor
    <JsonProperty("TipoIdentificacion")>
    Public Property TipoIdentificacion As String
    <JsonProperty("numeroIdentificacion")>
    Public Property numeroIdentificacion As String
End Class

Public Class Receptor
    <JsonProperty("TipoIdentificacion")>
    Public Property TipoIdentificacion As String
    <JsonProperty("numeroIdentificacion")>
    Public Property numeroIdentificacion As String
    Public Property sinReceptor As Boolean = False
End Class